msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md

```diff
@@ -168,7 +168,7 @@ The `GetParticipant` API returns the following response codes:
 | **403** | Get participant information isn't shared with the app. If the app isn't installed in the meeting, it triggers the most common error response 403. If the tenant admin disables or blocks the app during live site migration, 403 error response is triggered. |
 | **200** | The participant information is successfully retrieved.|
 | **401** | The app responds with an invalid token.|
-| **404** | The meeting has either expired or participant cannot be found.|
+| **404** | Either the meeting has expired or the participant cannot be found.|
```

Comment:
This changes the parallel structure with the other points. Please rephrase: The meeting has either expired or the participant can't be found.

Reply:
Hi, 

I have rephrased the sentence as you have suggested. However, the use of "either...or" is grammatically incorrect here. "Either...or" is used to emphasize a choice. It would have been correct if the sentence was something like "The meeting has either expired or canceled". 

Regards,
Pallabi

Reply:
Hi,

Keep it as: The meeting has expired or participant cannot be found.

Regards,
Abira

Reply:
Fixed. Thanks.